src/views/Records/Record.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div
+    class="standard container-fluid"
+  >
+    <div v-if="error"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['v-if' should be on a new line.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4965485)